package.json

```diff
@@ -23,10 +23,8 @@
     "type": "git",
     "url": "https://github.com/onefinestay/react-daterange-picker"
   },
-  "peerDependencies": {
-    "react": ">=0.12.0"
-  },
   "dependencies": {
+    "react": ">=0.12.0",
```

Comment:
`peerDependencies` are a bit of a mess (hopefully npm3 will sort them out) but in the mean time if we could keep React as a peer it means people won't be loading two different versions of React with is a whole other world of pain.